pkgs/development/compilers/lunarml/default.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, fetchFromGitHub
+, fetchzip
+, stdenvNoCC
+, mlton
+, lua5_3
+, luajit
+, nodejs
+, withExamples ? false
+}:
+
+let
+  pname = "lunarml";
+in
+stdenvNoCC.mkDerivation {
+  inherit pname;
+
+  version = "unstable-2023-06-25";
+
+  src = fetchFromGitHub {
+    owner = "minoki";
+    repo = "LunarML";
+    rev = "f58f90cf7a2f26340403245907ed183f6a12ab52";
+    sha256 = "djHJfUAPplsejFW9L3fbwTeeWgvR+gKkI8TmwIh8n7E=";
+  };
+
+  outputs = [ "doc" "out" ];
```

```suggestion
  outputs = [ "out" "doc" ];
```

Main output goes first